bamboo/deploy-dev-integration-test-stack.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
This script always runs in parallel with the unit/audit/lint jobs.    We want, however, to *not* run deployment if the flag is set to test from a published artifact.   In that case, wait until the publish job happens and then run deployment as part of the integration test step